interfaces/Julia/test/MOI_wrapper.jl

```diff
@@ -769,56 +960,103 @@ function test_vector_nonlinear_oracle_scalar_nonlinear_equivalent()
     @test ≈(MOI.get(model, MOI.VariablePrimal(), t), -log(sqrt(2)); atol)
     @test ≈(MOI.get(model, MOI.ConstraintPrimal(), c1), 0.0; atol)
     @test ≈(MOI.get(model, MOI.ConstraintPrimal(), c2), 0.0; atol)
-    @test ≈(MOI.get(model, MOI.ConstraintDual(), c1), -0.5; atol)
-    @test ≈(MOI.get(model, MOI.ConstraintDual(), c2), 1 / sqrt(2); atol)
-    @test ≈(MOI.get(model, MOI.ConstraintDual(), c_x3), -0.5; atol)
+    @test ≈(MOI.get(model, MOI.ConstraintDual(), c1), 0.5; atol)
+    @test ≈(MOI.get(model, MOI.ConstraintDual(), c2), -1 / sqrt(2); atol)
+    @test ≈(MOI.get(model, MOI.ConstraintDual(), c_x3), 0.0; atol)
```

Comment:
These are wrong, and it's pretty easy to demonstrate: the primal value of `x[3]` is `1.0`, which is binding for `c_x3`, so the dual should be negative.